arrows/pytorch/SRNN_tracking.py

```diff
@@ -71,29 +71,26 @@
 
 g_config = get_config()
 
-def print(msg):
-   import threading
-   import traceback
-   try:
-       msg = '[{}] {}'.format(threading.current_thread(), msg)
-   #    with open('database/Logs/SRNN_Tracking_Log', 'a') as f:
-   #        f.write(str(msg) + '\n')
-   except Exception as ex:
-       with open('database/Logs/SRNN_Tracking_Error_Log', 'a') as f:
-           f.write('Error durring print! Attempting to report\n')
-           f.write(repr(ex) + '\n')
-           f.write(traceback.format_exc() + '\n')
-           raise
+#def print(msg):
+   #import threading
+   #import traceback
+   #try:
+       #msg = '[{}] {}'.format(threading.current_thread(), msg)
+   ##    with open('database/Logs/SRNN_Tracking_Log', 'a') as f:
+   ##        f.write(str(msg) + '\n')
+   #except Exception as ex:
+       #with open('database/Logs/SRNN_Tracking_Error_Log', 'a') as f:
+           #f.write('Error durring print! Attempting to report\n')
+           #f.write(repr(ex) + '\n')
+           #f.write(traceback.format_exc() + '\n')
+           #raise
```

Comment:
It would be nice if we can somehow parametize these, unfortunately they're required for running the pipeline in the query GUI for now for a semi-unknown reason (the stdout pipe is NULL and writing to it causes an error). Fixing the later would probably be a better solution

Reply:
This particular change is already in `viame/master`. Do we want to revert it? (Also @dongshuhao)